docs/component-spec/modules.md

```diff
@@ -120,6 +120,14 @@ Origami modules are generally installed based on a semver range. To ensure new r
 * If releasing a module that contains generated content (fonts for example), make sure the new files are in the git repository (git ls-files) at the release commit
 * Don’t make a major release until all or most dependants have removed deprecated features
 
+#### Deprecation of a module
+
+In an event of deprecating a module within Origami, the following steps will be undertaken.
+
+1. Modify `origami.json` to change the `supportStatus` from `active` to `deprecated`.
```

Comment:
Just `to *deprecated*` would suffice here as it may have been another status before